docs/adrs/nnnn-aws-control-tower.md

```diff
@@ -0,0 +1,33 @@
+
+# Control Tower (CT) as foundation for a secure multi-account setup
+
+## Context
+
+AWS proposes Multi-Account architectures as best practise (see e.g. Well-Architected) framework, so superwerker is following this best practise.
+CT is the native service as the foundation for a secure multi-account. Since superwerker prefers to use native services whenever possible, CT is used.
+
+## Decision
+
+ - Since CT has no API yet, automate it via CloudWatch Synthetics (CWS) Canaries
+ - In order to keep it simple superwerker can be installed into regions which Control Tower supports. No multi-region support for now.
+ - Update Control Tower to latest version when installing/updating superwerker in order to ensure a sane state
+ - Since CT uses AWS SSO, it is set as the default Login and AWS Account switching solution.
+
+### Account Factory 
+
+ - superwerker promotes using the CTAF for enrolling/vending new Accounts
+
+## Consequences
+
+ - superwerker only supports regions supported by CT
+ - CT implies additional costs (e.g. Config Rules)
+ - Using CWS Canaries provides an audit log of the Click Ops because of the screenshots
+ - The CT LZ setup/update/repair has to be supervised since it fails sometimes and has to be restarted (TODO)
+ - superwerker provides links to how-tos for wiring existing IdPs like GSuite, AzureAD, etc. (TODO)\
+ 
+## TODO
+
+ - [ ] How to handle SSO and Users
+ - [ ] Document CW Events / SSM Parameters
+ - [ ] Should we enable additional GuardRails? Overlaps with SH?
+ - [ ] Should we consider activating/deactivating VPC creation?
```

Comment:
Decision: we know they are kinda useless and inconsistent (e.g. multi region setups) but we wait until we build a network feature for this.